tests/ui_tests/test_jobs_page.py

```diff
@@ -45,11 +46,10 @@
 SERVICE_NAME = 'test_service'
 COMPONENT_NAME = 'test_component'
 
-# pylint: disable=redefined-outer-name
+# pylint: disable=redefined-outer-name, unused-argument, no-self-use
```

Comment:
Why are you mute these?

Reply:
eg, [R0201(no-self-use), TestTaskPage.test_cluster_action_job] Method could be a function

'unused-argument' has already been there, I just moved in one place